py-pkg/pkgnet/create_package_report.py

```diff
@@ -0,0 +1,26 @@
+from pkgnet.abstract_graph_reporter import AbstractGraphReporter
+from pkgnet.dependency_reporter import DependencyReporter
+from pkgnet.function_reporter import FunctionReporter
+from pkgnet.package_report import PackageReport
+from pkgnet.summary_reporter import SummaryReporter
+
+
+def default_reporters():
+    return [SummaryReporter(), DependencyReporter(), FunctionReporter()]
+
+
+def create_package_report(pkg_name, report_path, pkg_reporters=None, pkg_path=None):
```

Comment:
I'm going to keep my comments limited to the API in this review, I think that's the most important thing for us to agree on.

I understand that `snake_case` is the dominant style for functions in Python, and I'm ok with this being `create_package_report()` in Python and `CreatePackageReport()` in R.

Other than that casing difference, would you and @bburns632 be ok with the following statements about the interfaces?

* order and casing of keyword arguments is the same (for functions of public methods that exist in both packages)
* if `**kwargs` is used in Python, `...`  is used in R
* everything we want "internal" in python uses the convention of a leading `_`

I don't actually care about whether we literally use `doppel-cli` to encforce this, but I want us to at least agree on what we will and won't keep the same.

Reply:
Those bullets seem reasonable

Reply:
This makes sense. I've gone through tried to make consistently named and show up in the same order. If I missed something, let me know.